<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Updater</class>
 <widget class="QWidget" name="Updater">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: white;
}</string>
  </property>
  <widget class="QLabel" name="lblInstalled">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Installed version: 1.00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblCaption">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: #2567c0;
}</string>
   </property>
   <property name="text">
    <string>Test Program</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/assets/close.png</normaloff>:/images/assets/close.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblLatest">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Latest version: 1.10</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUpdate">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>240</y>
     <width>160</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Liberation Sans</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border: 1px solid #cccccc;
    border-radius: 10px;
	color: #424242;
    background-color: white;
}

QPushButton:pressed {
	color: white;
    background-color: #2567c0;
}</string>
   </property>
   <property name="text">
    <string>Update Now</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblStatus">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>225</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: #c44512;
}</string>
   </property>
   <property name="text">
    <string>New version available!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCheck">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>161</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: #2975dc;
}</string>
   </property>
   <property name="text">
    <string>Check for updates</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
